tensorboard/compat/tensorflow_stub/io/gfile.py

```diff
@@ -366,6 +445,15 @@ def read(self, n=None):
 
         return result
 
+    def write(self, file_content):
```

Comment:
Please add a docstring here explaining that this always appends to the end of the file.

Reply:
I don't think that matches TensorFlow GFile semantics, but maybe I'm wrong? Seems like it should operate like open(filename, mode), which means it clears out the contents and then appends on each write. How about this for docstring:

```
        """Writes string file contents to file, clearing contents of the
        file on first write and then appending on subsequent calls.

        Args:
            file_content: string, the contents
        """
```

Reply:
Got it - thanks for the clarification.